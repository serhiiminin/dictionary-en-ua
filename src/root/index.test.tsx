import React from 'react';
import { shallow } from 'enzyme';
import Component from '.';

describe('Root', (): void => {
  let wrapper: object;

  beforeEach((): void => {
    wrapper = shallow(<Component />);
  });
  test('render', (): void => {
    expect(wrapper).toMatchSnapshot();
  });
});
