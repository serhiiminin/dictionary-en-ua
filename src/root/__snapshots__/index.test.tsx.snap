// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root render 1`] = `
<ThemeProvider
  theme={
    Object {
      "breakpoints": Object {
        "between": [Function],
        "down": [Function],
        "keys": Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ],
        "only": [Function],
        "up": [Function],
        "values": Object {
          "lg": 1280,
          "md": 960,
          "sm": 600,
          "xl": 1920,
          "xs": 0,
        },
        "width": [Function],
      },
      "direction": "ltr",
      "main": Object {
        "borderRadius": Object {
          "lg": "1.2rem",
          "md": ".8rem",
          "sm": ".4rem",
          "xl": "1.6rem",
          "xs": ".2rem",
        },
        "borderStyle": Object {
          "base": "solid",
        },
        "borderWidth": Object {
          "base": ".1rem",
        },
        "boxShadow": Object {
          "block": ".4rem .8rem 4rem rgba(123, 123, 123, 0.4)",
        },
        "breakpoint": Object {
          "lg": "120em",
          "md": "99.2em",
          "sm": "76.8em",
          "xs": "57.6em",
        },
        "color": Object {
          "background": "#FFFFFF",
          "contrastText": "#AC3B61",
          "dark": "#86D1FF",
          "gradient": Object {
            "block": "linear-gradient(149.02deg, #86d1ff -11.95%, #bbd6fd 89.7%, #c5d7fd 89.71%, #c5d7fd 89.73%)",
          },
          "light": "#C5D7FD",
          "main": "#979797",
          "notification": Object {
            "error": "#F21600",
            "info": "#A9D9E4",
            "success": "#83F570",
            "warning": "#FAE639",
          },
          "text": "#161617",
        },
        "fontFamily": Object {
          "cairoBold": "Cairo Bold",
          "cairoRegular": "Cairo Regular",
        },
        "fontSize": Object {
          "lg": "1.8rem",
          "md": "1.6rem",
          "sm": "1.4rem",
          "xl": "2.4rem",
          "xs": "1.2rem",
        },
        "letterSpacing": Object {
          "lg": ".8rem",
          "md": ".6rem",
          "sm": ".4rem",
          "xl": "1rem",
          "xs": ".2rem",
        },
        "lineHeight": Object {
          "lg": ".8rem",
          "md": ".6rem",
          "sm": ".4rem",
          "xl": "1rem",
          "xs": ".2rem",
        },
        "opacity": Object {
          "disabled": 0.7,
          "snackbar": 0.9,
        },
        "space": Object {
          "lg": "2rem",
          "md": "1.5rem",
          "sm": "1rem",
          "xl": "2.5rem",
          "xs": ".5rem",
        },
        "transition": Object {
          "base": 300,
          "button": 100,
          "controls": 100,
          "notification": 300,
        },
        "zIndex": Object {
          "notification": 1000,
        },
      },
      "mixins": Object {
        "gutters": [Function],
        "toolbar": Object {
          "@media (min-width:0px) and (orientation: landscape)": Object {
            "minHeight": 48,
          },
          "@media (min-width:600px)": Object {
            "minHeight": 64,
          },
          "minHeight": 56,
        },
      },
      "overrides": Object {
        "MuiButton": Object {
          "outlinedPrimary": Object {
            "borderColor": "#161617",
            "color": "#161617",
          },
          "root": Object {
            "&$contained&$disabled": Object {
              "color": "#FFFFFF",
            },
            "&$disabled": Object {
              "opacity": 0.7,
            },
            "&:hover": Object {
              "background": "#979797",
              "opacity": 0.7,
            },
            "borderRadius": ".8rem",
            "color": "#161617",
            "textTransform": "none",
            "transition": 300,
          },
        },
        "MuiChip": Object {
          "root": Object {
            "background": "transparent",
            "border": ".1rem solid #161617",
            "color": "#161617",
            "fontSize": "1.4rem",
            "opacity": 0.7,
          },
        },
        "MuiFormHelperText": Object {
          "root": Object {
            "&$contained": Object {
              "margin": ".5rem 1rem 0",
            },
            "&$error": Object {
              "color": "#F21600",
            },
            "fontSize": "1.2rem",
          },
        },
        "MuiInputLabel": Object {
          "outlined": Object {
            "transform": "translate(2.5rem, 2rem) scale(1)",
          },
          "root": Object {
            "&$focused": Object {
              "color": "#979797",
            },
            "fontSize": "1.4rem",
          },
        },
        "MuiLinearProgress": Object {
          "root": Object {
            "background": "#FFFFFF",
          },
        },
        "MuiMenuItem": Object {
          "root": Object {
            "&$selected": Object {
              "background": "#979797",
              "color": "#FFFFFF",
            },
            "&:hover": Object {
              "background": "#979797",
              "color": "#FFFFFF",
            },
            "width": "100%",
          },
        },
        "MuiOutlinedInput": Object {
          "input": Object {
            "padding": "2rem 2.5rem",
          },
          "notchedOutline": Object {
            "borderRadius": ".8rem",
          },
          "root": Object {
            "& $notchedOutline": Object {
              "borderColor": "#979797",
            },
            "&$focused $notchedOutline": Object {
              "borderWidth": 1,
            },
            "&:hover:not($disabled):not($focused):not($error) $notchedOutline": Object {
              "borderColor": "#979797",
            },
            "fontSize": "1.4rem",
            "transition": 300,
          },
        },
        "MuiSelect": Object {
          "root": Object {
            "width": "100%",
          },
          "select": Object {
            "background": "transparent !important",
          },
        },
        "MuiSnackbar": Object {
          "anchorOriginTopRight": Object {
            "opacity": 0.9,
          },
        },
        "MuiSnackbarContent": Object {
          "message": Object {
            "fontSize": "1.4rem",
          },
        },
        "MuiSvgIcon": Object {
          "root": Object {
            "&:hover": Object {
              "cursor": "pointer",
            },
            "fontSize": "1.8rem",
          },
        },
        "MuiTooltip": Object {
          "tooltipPlacementRight": Object {
            "background": "#161617",
            "color": "#FFFFFF",
            "fontSize": "1.2rem",
          },
        },
        "MuiTouchRipple": Object {
          "childPulsate": Object {
            "background": "#86D1FF",
          },
        },
      },
      "palette": Object {
        "action": Object {
          "active": "rgba(0, 0, 0, 0.54)",
          "disabled": "rgba(0, 0, 0, 0.26)",
          "disabledBackground": "rgba(0, 0, 0, 0.12)",
          "hover": "rgba(0, 0, 0, 0.08)",
          "hoverOpacity": 0.08,
          "selected": "rgba(0, 0, 0, 0.14)",
        },
        "augmentColor": [Function],
        "background": Object {
          "default": "#fafafa",
          "paper": "#FFFFFF",
        },
        "common": Object {
          "black": "#000",
          "white": "#fff",
        },
        "contrastThreshold": 3,
        "divider": "rgba(0, 0, 0, 0.12)",
        "error": Object {
          "contrastText": "#fff",
          "dark": "#d32f2f",
          "light": "#e57373",
          "main": "#f44336",
        },
        "getContrastText": [Function],
        "grey": Object {
          "100": "#f5f5f5",
          "200": "#eeeeee",
          "300": "#e0e0e0",
          "400": "#bdbdbd",
          "50": "#fafafa",
          "500": "#9e9e9e",
          "600": "#757575",
          "700": "#616161",
          "800": "#424242",
          "900": "#212121",
          "A100": "#d5d5d5",
          "A200": "#aaaaaa",
          "A400": "#303030",
          "A700": "#616161",
        },
        "primary": Object {
          "contrastText": "#AC3B61",
          "dark": "#86D1FF",
          "light": "#C5D7FD",
          "main": "#979797",
        },
        "secondary": Object {
          "contrastText": "#AC3B61",
          "dark": "#86D1FF",
          "light": "#C5D7FD",
          "main": "#979797",
        },
        "text": Object {
          "disabled": "rgba(0, 0, 0, 0.38)",
          "hint": "rgba(0, 0, 0, 0.38)",
          "primary": "#161617",
          "secondary": "rgba(0, 0, 0, 0.54)",
        },
        "tonalOffset": 0.2,
        "type": "light",
      },
      "props": Object {},
      "shadows": Array [
        "none",
        "0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12)",
        "0px 1px 5px 0px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 3px 1px -2px rgba(0,0,0,0.12)",
        "0px 1px 8px 0px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 3px 3px -2px rgba(0,0,0,0.12)",
        "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
        "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
        "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
        "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
        "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
        "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
        "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
        "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
        "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
        "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
        "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
        "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
        "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
        "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
        "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
        "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
        "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
        "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
      ],
      "shape": Object {
        "borderRadius": 4,
      },
      "spacing": [Function],
      "transitions": Object {
        "create": [Function],
        "duration": Object {
          "complex": 375,
          "enteringScreen": 225,
          "leavingScreen": 195,
          "short": 250,
          "shorter": 200,
          "shortest": 150,
          "standard": 300,
        },
        "easing": Object {
          "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
          "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
          "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
          "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
        },
        "getAutoHeightDuration": [Function],
      },
      "typography": Object {
        "body1": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1.1428571428571428rem",
          "fontWeight": 400,
          "lineHeight": 1.5,
        },
        "body2": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1rem",
          "fontWeight": 400,
          "lineHeight": 1.43,
        },
        "button": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1rem",
          "fontWeight": 500,
          "lineHeight": 1.75,
          "textTransform": "uppercase",
        },
        "caption": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "0.8571428571428571rem",
          "fontWeight": 400,
          "lineHeight": 1.66,
        },
        "fontFamily": "Cairo Regular, sans-serif",
        "fontSize": 16,
        "fontWeightBold": 700,
        "fontWeightLight": 300,
        "fontWeightMedium": 500,
        "fontWeightRegular": 400,
        "h1": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "6.857142857142857rem",
          "fontWeight": 300,
          "lineHeight": 1,
        },
        "h2": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "4.285714285714286rem",
          "fontWeight": 300,
          "lineHeight": 1,
        },
        "h3": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "3.4285714285714284rem",
          "fontWeight": 400,
          "lineHeight": 1.04,
        },
        "h4": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "2.4285714285714284rem",
          "fontWeight": 400,
          "lineHeight": 1.17,
        },
        "h5": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1.7142857142857142rem",
          "fontWeight": 400,
          "lineHeight": 1.33,
        },
        "h6": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1.4285714285714284rem",
          "fontWeight": 500,
          "lineHeight": 1.6,
        },
        "htmlFontSize": 16,
        "overline": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "0.8571428571428571rem",
          "fontWeight": 400,
          "lineHeight": 2.66,
          "textTransform": "uppercase",
        },
        "pxToRem": [Function],
        "round": [Function],
        "subtitle1": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1.1428571428571428rem",
          "fontWeight": 400,
          "lineHeight": 1.75,
        },
        "subtitle2": Object {
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": "1rem",
          "fontWeight": 500,
          "lineHeight": 1.57,
        },
      },
      "zIndex": Object {
        "appBar": 1100,
        "drawer": 1200,
        "mobileStepper": 1000,
        "modal": 1300,
        "snackbar": 1400,
        "tooltip": 1500,
      },
    }
  }
>
  <ThemeProvider
    theme={
      Object {
        "breakpoints": Object {
          "between": [Function],
          "down": [Function],
          "keys": Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "only": [Function],
          "up": [Function],
          "values": Object {
            "lg": 1280,
            "md": 960,
            "sm": 600,
            "xl": 1920,
            "xs": 0,
          },
          "width": [Function],
        },
        "direction": "ltr",
        "main": Object {
          "borderRadius": Object {
            "lg": "1.2rem",
            "md": ".8rem",
            "sm": ".4rem",
            "xl": "1.6rem",
            "xs": ".2rem",
          },
          "borderStyle": Object {
            "base": "solid",
          },
          "borderWidth": Object {
            "base": ".1rem",
          },
          "boxShadow": Object {
            "block": ".4rem .8rem 4rem rgba(123, 123, 123, 0.4)",
          },
          "breakpoint": Object {
            "lg": "120em",
            "md": "99.2em",
            "sm": "76.8em",
            "xs": "57.6em",
          },
          "color": Object {
            "background": "#FFFFFF",
            "contrastText": "#AC3B61",
            "dark": "#86D1FF",
            "gradient": Object {
              "block": "linear-gradient(149.02deg, #86d1ff -11.95%, #bbd6fd 89.7%, #c5d7fd 89.71%, #c5d7fd 89.73%)",
            },
            "light": "#C5D7FD",
            "main": "#979797",
            "notification": Object {
              "error": "#F21600",
              "info": "#A9D9E4",
              "success": "#83F570",
              "warning": "#FAE639",
            },
            "text": "#161617",
          },
          "fontFamily": Object {
            "cairoBold": "Cairo Bold",
            "cairoRegular": "Cairo Regular",
          },
          "fontSize": Object {
            "lg": "1.8rem",
            "md": "1.6rem",
            "sm": "1.4rem",
            "xl": "2.4rem",
            "xs": "1.2rem",
          },
          "letterSpacing": Object {
            "lg": ".8rem",
            "md": ".6rem",
            "sm": ".4rem",
            "xl": "1rem",
            "xs": ".2rem",
          },
          "lineHeight": Object {
            "lg": ".8rem",
            "md": ".6rem",
            "sm": ".4rem",
            "xl": "1rem",
            "xs": ".2rem",
          },
          "opacity": Object {
            "disabled": 0.7,
            "snackbar": 0.9,
          },
          "space": Object {
            "lg": "2rem",
            "md": "1.5rem",
            "sm": "1rem",
            "xl": "2.5rem",
            "xs": ".5rem",
          },
          "transition": Object {
            "base": 300,
            "button": 100,
            "controls": 100,
            "notification": 300,
          },
          "zIndex": Object {
            "notification": 1000,
          },
        },
        "mixins": Object {
          "gutters": [Function],
          "toolbar": Object {
            "@media (min-width:0px) and (orientation: landscape)": Object {
              "minHeight": 48,
            },
            "@media (min-width:600px)": Object {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "overrides": Object {
          "MuiButton": Object {
            "outlinedPrimary": Object {
              "borderColor": "#161617",
              "color": "#161617",
            },
            "root": Object {
              "&$contained&$disabled": Object {
                "color": "#FFFFFF",
              },
              "&$disabled": Object {
                "opacity": 0.7,
              },
              "&:hover": Object {
                "background": "#979797",
                "opacity": 0.7,
              },
              "borderRadius": ".8rem",
              "color": "#161617",
              "textTransform": "none",
              "transition": 300,
            },
          },
          "MuiChip": Object {
            "root": Object {
              "background": "transparent",
              "border": ".1rem solid #161617",
              "color": "#161617",
              "fontSize": "1.4rem",
              "opacity": 0.7,
            },
          },
          "MuiFormHelperText": Object {
            "root": Object {
              "&$contained": Object {
                "margin": ".5rem 1rem 0",
              },
              "&$error": Object {
                "color": "#F21600",
              },
              "fontSize": "1.2rem",
            },
          },
          "MuiInputLabel": Object {
            "outlined": Object {
              "transform": "translate(2.5rem, 2rem) scale(1)",
            },
            "root": Object {
              "&$focused": Object {
                "color": "#979797",
              },
              "fontSize": "1.4rem",
            },
          },
          "MuiLinearProgress": Object {
            "root": Object {
              "background": "#FFFFFF",
            },
          },
          "MuiMenuItem": Object {
            "root": Object {
              "&$selected": Object {
                "background": "#979797",
                "color": "#FFFFFF",
              },
              "&:hover": Object {
                "background": "#979797",
                "color": "#FFFFFF",
              },
              "width": "100%",
            },
          },
          "MuiOutlinedInput": Object {
            "input": Object {
              "padding": "2rem 2.5rem",
            },
            "notchedOutline": Object {
              "borderRadius": ".8rem",
            },
            "root": Object {
              "& $notchedOutline": Object {
                "borderColor": "#979797",
              },
              "&$focused $notchedOutline": Object {
                "borderWidth": 1,
              },
              "&:hover:not($disabled):not($focused):not($error) $notchedOutline": Object {
                "borderColor": "#979797",
              },
              "fontSize": "1.4rem",
              "transition": 300,
            },
          },
          "MuiSelect": Object {
            "root": Object {
              "width": "100%",
            },
            "select": Object {
              "background": "transparent !important",
            },
          },
          "MuiSnackbar": Object {
            "anchorOriginTopRight": Object {
              "opacity": 0.9,
            },
          },
          "MuiSnackbarContent": Object {
            "message": Object {
              "fontSize": "1.4rem",
            },
          },
          "MuiSvgIcon": Object {
            "root": Object {
              "&:hover": Object {
                "cursor": "pointer",
              },
              "fontSize": "1.8rem",
            },
          },
          "MuiTooltip": Object {
            "tooltipPlacementRight": Object {
              "background": "#161617",
              "color": "#FFFFFF",
              "fontSize": "1.2rem",
            },
          },
          "MuiTouchRipple": Object {
            "childPulsate": Object {
              "background": "#86D1FF",
            },
          },
        },
        "palette": Object {
          "action": Object {
            "active": "rgba(0, 0, 0, 0.54)",
            "disabled": "rgba(0, 0, 0, 0.26)",
            "disabledBackground": "rgba(0, 0, 0, 0.12)",
            "hover": "rgba(0, 0, 0, 0.08)",
            "hoverOpacity": 0.08,
            "selected": "rgba(0, 0, 0, 0.14)",
          },
          "augmentColor": [Function],
          "background": Object {
            "default": "#fafafa",
            "paper": "#FFFFFF",
          },
          "common": Object {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(0, 0, 0, 0.12)",
          "error": Object {
            "contrastText": "#fff",
            "dark": "#d32f2f",
            "light": "#e57373",
            "main": "#f44336",
          },
          "getContrastText": [Function],
          "grey": Object {
            "100": "#f5f5f5",
            "200": "#eeeeee",
            "300": "#e0e0e0",
            "400": "#bdbdbd",
            "50": "#fafafa",
            "500": "#9e9e9e",
            "600": "#757575",
            "700": "#616161",
            "800": "#424242",
            "900": "#212121",
            "A100": "#d5d5d5",
            "A200": "#aaaaaa",
            "A400": "#303030",
            "A700": "#616161",
          },
          "primary": Object {
            "contrastText": "#AC3B61",
            "dark": "#86D1FF",
            "light": "#C5D7FD",
            "main": "#979797",
          },
          "secondary": Object {
            "contrastText": "#AC3B61",
            "dark": "#86D1FF",
            "light": "#C5D7FD",
            "main": "#979797",
          },
          "text": Object {
            "disabled": "rgba(0, 0, 0, 0.38)",
            "hint": "rgba(0, 0, 0, 0.38)",
            "primary": "#161617",
            "secondary": "rgba(0, 0, 0, 0.54)",
          },
          "tonalOffset": 0.2,
          "type": "light",
        },
        "props": Object {},
        "shadows": Array [
          "none",
          "0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12)",
          "0px 1px 5px 0px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 3px 1px -2px rgba(0,0,0,0.12)",
          "0px 1px 8px 0px rgba(0,0,0,0.2),0px 3px 4px 0px rgba(0,0,0,0.14),0px 3px 3px -2px rgba(0,0,0,0.12)",
          "0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12)",
          "0px 3px 5px -1px rgba(0,0,0,0.2),0px 5px 8px 0px rgba(0,0,0,0.14),0px 1px 14px 0px rgba(0,0,0,0.12)",
          "0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12)",
          "0px 4px 5px -2px rgba(0,0,0,0.2),0px 7px 10px 1px rgba(0,0,0,0.14),0px 2px 16px 1px rgba(0,0,0,0.12)",
          "0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12)",
          "0px 5px 6px -3px rgba(0,0,0,0.2),0px 9px 12px 1px rgba(0,0,0,0.14),0px 3px 16px 2px rgba(0,0,0,0.12)",
          "0px 6px 6px -3px rgba(0,0,0,0.2),0px 10px 14px 1px rgba(0,0,0,0.14),0px 4px 18px 3px rgba(0,0,0,0.12)",
          "0px 6px 7px -4px rgba(0,0,0,0.2),0px 11px 15px 1px rgba(0,0,0,0.14),0px 4px 20px 3px rgba(0,0,0,0.12)",
          "0px 7px 8px -4px rgba(0,0,0,0.2),0px 12px 17px 2px rgba(0,0,0,0.14),0px 5px 22px 4px rgba(0,0,0,0.12)",
          "0px 7px 8px -4px rgba(0,0,0,0.2),0px 13px 19px 2px rgba(0,0,0,0.14),0px 5px 24px 4px rgba(0,0,0,0.12)",
          "0px 7px 9px -4px rgba(0,0,0,0.2),0px 14px 21px 2px rgba(0,0,0,0.14),0px 5px 26px 4px rgba(0,0,0,0.12)",
          "0px 8px 9px -5px rgba(0,0,0,0.2),0px 15px 22px 2px rgba(0,0,0,0.14),0px 6px 28px 5px rgba(0,0,0,0.12)",
          "0px 8px 10px -5px rgba(0,0,0,0.2),0px 16px 24px 2px rgba(0,0,0,0.14),0px 6px 30px 5px rgba(0,0,0,0.12)",
          "0px 8px 11px -5px rgba(0,0,0,0.2),0px 17px 26px 2px rgba(0,0,0,0.14),0px 6px 32px 5px rgba(0,0,0,0.12)",
          "0px 9px 11px -5px rgba(0,0,0,0.2),0px 18px 28px 2px rgba(0,0,0,0.14),0px 7px 34px 6px rgba(0,0,0,0.12)",
          "0px 9px 12px -6px rgba(0,0,0,0.2),0px 19px 29px 2px rgba(0,0,0,0.14),0px 7px 36px 6px rgba(0,0,0,0.12)",
          "0px 10px 13px -6px rgba(0,0,0,0.2),0px 20px 31px 3px rgba(0,0,0,0.14),0px 8px 38px 7px rgba(0,0,0,0.12)",
          "0px 10px 13px -6px rgba(0,0,0,0.2),0px 21px 33px 3px rgba(0,0,0,0.14),0px 8px 40px 7px rgba(0,0,0,0.12)",
          "0px 10px 14px -6px rgba(0,0,0,0.2),0px 22px 35px 3px rgba(0,0,0,0.14),0px 8px 42px 7px rgba(0,0,0,0.12)",
          "0px 11px 14px -7px rgba(0,0,0,0.2),0px 23px 36px 3px rgba(0,0,0,0.14),0px 9px 44px 8px rgba(0,0,0,0.12)",
          "0px 11px 15px -7px rgba(0,0,0,0.2),0px 24px 38px 3px rgba(0,0,0,0.14),0px 9px 46px 8px rgba(0,0,0,0.12)",
        ],
        "shape": Object {
          "borderRadius": 4,
        },
        "spacing": [Function],
        "transitions": Object {
          "create": [Function],
          "duration": Object {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": Object {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": Object {
          "body1": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1.1428571428571428rem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "body2": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1rem",
            "fontWeight": 400,
            "lineHeight": 1.43,
          },
          "button": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1rem",
            "fontWeight": 500,
            "lineHeight": 1.75,
            "textTransform": "uppercase",
          },
          "caption": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "0.8571428571428571rem",
            "fontWeight": 400,
            "lineHeight": 1.66,
          },
          "fontFamily": "Cairo Regular, sans-serif",
          "fontSize": 16,
          "fontWeightBold": 700,
          "fontWeightLight": 300,
          "fontWeightMedium": 500,
          "fontWeightRegular": 400,
          "h1": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "6.857142857142857rem",
            "fontWeight": 300,
            "lineHeight": 1,
          },
          "h2": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "4.285714285714286rem",
            "fontWeight": 300,
            "lineHeight": 1,
          },
          "h3": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "3.4285714285714284rem",
            "fontWeight": 400,
            "lineHeight": 1.04,
          },
          "h4": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "2.4285714285714284rem",
            "fontWeight": 400,
            "lineHeight": 1.17,
          },
          "h5": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1.7142857142857142rem",
            "fontWeight": 400,
            "lineHeight": 1.33,
          },
          "h6": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1.4285714285714284rem",
            "fontWeight": 500,
            "lineHeight": 1.6,
          },
          "htmlFontSize": 16,
          "overline": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "0.8571428571428571rem",
            "fontWeight": 400,
            "lineHeight": 2.66,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "round": [Function],
          "subtitle1": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1.1428571428571428rem",
            "fontWeight": 400,
            "lineHeight": 1.75,
          },
          "subtitle2": Object {
            "fontFamily": "Cairo Regular, sans-serif",
            "fontSize": "1rem",
            "fontWeight": 500,
            "lineHeight": 1.57,
          },
        },
        "zIndex": Object {
          "appBar": 1100,
          "drawer": 1200,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "tooltip": 1500,
        },
      }
    }
  >
    <GlobalStyleComponent />
    <BrowserRouter
      basename=""
    >
      <SnackbarProvider
        TransitionComponent={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "propTypes": Object {
              "children": [Function],
              "in": [Function],
              "onEnter": [Function],
              "onExit": [Function],
              "style": [Function],
              "timeout": [Function],
            },
            "render": [Function],
          }
        }
        anchorOrigin={
          Object {
            "horizontal": "left",
            "vertical": "bottom",
          }
        }
        autoHideDuration={5000}
        dense={false}
        hideIconVariant={false}
        iconVariant={Object {}}
        maxSnack={5}
        preventDuplicate={true}
      >
        <StateProvider>
          <Component />
          <Switch>
            <Route
              component={[Function]}
              exact={true}
              path="/"
            />
            <Route
              component={[Function]}
              path="/auth"
            />
            <Component
              component={[Function]}
              path="/words"
            />
            <Route
              component={[Function]}
            />
          </Switch>
        </StateProvider>
      </SnackbarProvider>
    </BrowserRouter>
  </ThemeProvider>
</ThemeProvider>
`;
